app/controllers/submissions_controller.rb

```diff
@@ -71,9 +71,10 @@ def submission_params
       :max_processes_and_or_threads,
       :enable_per_process_and_thread_time_limit,
       :enable_per_process_and_thread_memory_limit,
-      :max_file_size
+      :max_file_size,
+      { webhooks: [ :url, postdata: {} ] },
```

Comment:
Why does the user need to send `postdata`? Why not just array of urls?

Reply:
Our use case requires passing context associated with the job here. So postdata is abstract here and can be used to pass tokens or any other job context.